netsim/providers/libvirt.yml

```diff
@@ -41,3 +41,6 @@ attributes:
     uplink: str
   global:
     providers:
+
+inventory:
+  ansible_ssh_private_key_file: .vagrant/machines/{ name }/libvirt/private_key
```

Comment:
This is set for **all** Vagrant devices and probably breaks **every single device that uses Vagrant default key** (as in: most of them, I did a spot check on Cisco IOSv). Awesome. Just awesome.

Would you once in a while stop for a microsecond, think about the wider implications of your changes, and make the minimum amount of changes necessary?